src/example-types/metricGroups/pivot.js

```diff
@@ -273,6 +297,16 @@ let createPivotScope = (node, schema, getStats) => {
       async (children, group, index, groups) => {
         // Defaulting the group size to be 10
         if (!group.size) group.size = 10
+        // We are iterating through the groups reversed so we need to subtract instead of add to get the right index
+        let reversedLookupIndex = groups.length - index - 1
+        let drilldownKey = _.get(
+          [groupingType, 'drilldown', reversedLookupIndex],
+          request
+        )
+        let groupResult = filterGroupRanges({
```

Comment:
Do you think it makes to keep it as `group`
```
group = filterGroupRanges(...
```
since we are already mutating it above and `result` is already an overloaded term in context of pivot

Reply:
I was on the fence about that same thing, but if it makes more sense to you that way I'm good with just re-assigning the variable

Reply:
@dshishkov @re1ro I re-assigned it.